fastlane/lib/fastlane/action.rb

```diff
@@ -149,5 +149,23 @@ def self.other_action
     def self.deprecated_notes
       nil
     end
+
+    def self.is_compatible?(operating_system, action_compatibility = {})
+      # TODO find way to motivate developer to do so implement `is_compatible` in action
```

Comment:
We could probably increase plugin rating if this is defined ^

Reply:
That's a great idea.